source/brailleDisplayDrivers/baum.py

```diff
@@ -329,8 +329,10 @@ class InputGesture(braille.BrailleDisplayGesture, brailleInput.BrailleInputGestu
 
 	source = BrailleDisplayDriver.name
 
-	def __init__(self, keysDown):
+	def __init__(self, model, keysDown):
 		super(InputGesture, self).__init__()
+		# Model identifiers should not contain spaces.
+		self.model = model.replace(" ", "")

```

Comment:
why removing spaces and not colons ':'?
Would that not confuse your regular expression?

A dot '.' would probably be less an issue as the model name follows after the new introduced dot? (I am not that familiar with regular expressions....). But at least in the source part before the dot, you probably need also to remove dots now.

Reply:
I don't expect dots and colon in models name here, only spaces, and as the doc string for _get_model says on BrailleDisplayGesture, this must be a name without spaces.

Reply:
That is an expectation; where is it enforced?
I still believe that a more restrictive regular expression should go hand in hand with: i) updating that doc string with a more strict requirement, and ii) an accordding sanitasation of the input.